tests/expectations/tests/objc_class.rs

```diff
@@ -9,6 +9,11 @@
 extern crate objc;
 #[allow(non_camel_case_types)]
 pub type id = *mut objc::runtime::Object;
+type Class = *mut objc::runtime::Class;
+#[inline]
+fn class(name: &str) -> Class {
+    unsafe { std::mem::transmute(objc::runtime::Class::get(name)) }
```

Comment:
But actually, it seems it's not needed, right? I mean, instead of having this function, why can't we just use something like the following in the methods?

```rust
unsafe fn method() {
    msg_send![objc::runtime::Class::get("ClassName").expect("Couldn't find ClassName"), ...] 
}
```